components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/APIMappingUtil.java

```diff
@@ -306,14 +306,14 @@ public static API fromDTOtoAPI(APIDTO dto, String provider) throws APIManagement
             model.setAccessControl(APIConstants.API_RESTRICTED_VISIBILITY);
         }
 
-        List<APIInfoAdditionalPropertiesDTO> additionalProperties = dto.getAdditionalProperties();
```

Comment:
Let's keep both implementations. otherwse onprem will not work

Reply:
Fixed with 68d90f7